scripts/build-ui.sh

```diff
@@ -66,5 +81,5 @@ git pull --ff-only origin "${UI_COMMITISH}"
 git reset --hard "${UI_COMMITISH}"
 
 yarn install
-yarn build
+EDITION=${UI_EDITION} yarn build
```

Comment:
This is to fix an issue where if you build it locally from cloning the repository, we weren't actually setting the edition of the UI correctly.

Reply:
Good catch!